cumulusci.yml

```diff
@@ -63,6 +76,14 @@ flows:
         steps:
             3:
                 task: load_dataset
+            4:
+                task: deploy_qa_config
+            5:
+                task: create_perms_testing_user
+            6:
+                task: assign_permission_sets
+                options:
+                    api_names: Disbursements
```

Comment:
This can be removed since you added `"permsets": ["Disbursements"]` to the user definition file.
```suggestion
```